### DISCUSSION TITLE
About components use cases

### DISCUSSION THREAD

**Discussion by vasucp1207** [tamagui/tamagui#1621] [General]
Currently, we see that for a component there is only a single use case as an example in the docs, Is it intentional or do you guys want to add more use cases in the docs in the future and extend components functionality more?

**Comment by natew** [comment]
We definitely want to add more examples, and if you're willing we can pay for that.

**Reply by vasucp1207** [reply]
Yeah, I'm willing to contribute to it also see the discussion on discord about pro-components.
